apps/web/src/components/Messages/Preview.tsx

```diff
@@ -82,10 +82,8 @@ const Preview: FC<PreviewProps> = ({
             <div className="flex justify-between space-x-1">
               <div className="flex items-center gap-1 overflow-hidden">
                 <div className="text-md truncate">
-                  {profile?.name
-                    ? sanitizeDisplayName(profile?.name) ??
-                      formatHandle(profile.handle)
-                    : ensName ?? formatAddress(conversationKey?.split('/')[0])}
+                  {sanitizeDisplayName(profile?.name) ??
+                    formatHandle(profile?.handle)}
```

Comment:
this removed ens name, how it will fix it?

Reply:
Ens name  is undefined, and you can get it from the profile object. This will fix the issue.